### ISSUE TITLE
Dragon 13 cannot select or dictate into number fields

### ISSUE THREAD

**Issue by selfthinker** [alphagov/reported-bugs#34]
**Upstream bug:** [Ref no 170922-000443](http://nuance.custhelp.com/app/account/questions/detail/i_id/2940826) (not public!)
**Date:** 2017-09-20 (?)
**Reported by:** Anika Henke
**Related to:** https://github.com/alphagov/govuk-frontend/issues/1101

---

### Overview

The bug report can be accessed with the GDS internal "nuance" google group account and can only be seen when you're logged in with that user.

The content of the bug report is:

> You can neither select any number fields (i.e. inputs with the type 'number') nor dictate into them with Dragon 13 in IE 11, or latest Chrome or Firefox.
(The selection obviously works with mouse grid, and the input works with the Dictation Box.)
You can try it out in this reduced test case: http://jsbin.com/munuvub

**Comment by selfthinker** [comment]
We only got an automated response from Nuance:

> Dear Accessibility Team,
> 
> Thank you for providing feedback about your Nuance product.
> 
> Your bug report will be reviewed for reproducibility
> Bugs reported will be triaged for fixing in a future update or version release of the product
> No response will be provided regarding your bug report
> If you require a response from Nuance, please open a new question and select a Support category
> Support categories include: Error Message, Hardware - Setup/Config, How to use Product, Product - > Problem/Issue, Product - Setup/Config
> We appreciate your feedback and help toward improving our products.
> 
> Best Regards
> 
> Desktop Support & Service
> Nuance Communication Inc.

So, it looks like they never give proper feedback to reports and we will never know how and if the status of the report progresses (unless it gets fixed and ends up in their public changelog).

**Comment by hannalaakso** [comment]
I've just tested this in IE11 and Chrome with Dragon 15 with the [reduced test case](https://output.jsbin.com/munuvub).

I can replicate the issue in both browsers:
- IE11: cannot select or dictate number field
- Chrome 79: can select number field but cannot dictate

**Comment by mfairchild365** [comment]
See also https://a11ysupport.io/tests/tech__html__input__input-number for related test results.

**Comment by selfthinker** [comment]
As part of the communication around #35 I was told by Nuance that:

> Dragon is not compatible with web pages and web based services, this is not supported!

I think we can close this as "won't fix" (and any other Dragon issue we might find).

**Comment by JamesCatt** [comment]
FYI it seems this has been fixed in Dragon 15. Verified by my own testing and also a11ysupport: https://a11ysupport.io/tests/tech__html__input__input-number#support-summary-by-at-vc

**Comment by 36degrees** [comment]
Closing on the basis of the above comment – we can reopen if we find it's still an issue.
